zipkin/src/test/java/com/linecorp/armeria/it/tracing/HttpTracingIntegrationTest.java

```diff
@@ -275,6 +275,37 @@ public void testClientInitiatedTrace() throws Exception {
 
         // Check the span names.
         assertThat(spans).allMatch(s -> "hello".equals(s.name()));
+
+        // Check wire times
+        long clientStartTime = clientFooSpan.timestampAsLong();
+        long clientWireSendTime = clientFooSpan.annotations().stream().filter(a -> a.value().equals("ws"))
+                                               .findFirst().get().timestamp();
+        long clientWireReceiveTime = clientFooSpan.annotations().stream().filter(a -> a.value().equals("wr"))
+                                               .findFirst().get().timestamp();
+        long clientEndTime = clientStartTime + clientFooSpan.durationAsLong();
+
+        long serverStartTime = serviceFooSpan.timestampAsLong();
+        long serverWireSendTime = serviceFooSpan.annotations().stream().filter(a -> a.value().equals("ws"))
+                                                .findFirst().get().timestamp();
+        long serverWireReceiveTime = serviceFooSpan.annotations().stream().filter(a -> a.value().equals("wr"))
+                                                   .findFirst().get().timestamp();
+        long serverEndTime = serverStartTime + serviceFooSpan.durationAsLong();
```

Comment:
Could these variables final?

Reply:
Ah didn't know we did that in tests too.